scripts/travis-deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Deploy to staging/production on master/release merges (not PRs)
+
+set -e
+
+# Don't deploy on PRs
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+  exit 0
+fi
+
+if [ "$TRAVIS_BRANCH" == "master" ]; then
+  # Deploy to staging on a merge to master
+  ember deploy -e staging
+elif [ "$TRAVIS_TAG" != "false" ]; then
```

Comment:
Nevermind, apparently `TRAVIS_BRANCH` is not set on the build running for a tag.